content/en/docs/Sample_Apps/arrive.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Arrive"
```

Comment:
Same thing with @‎rrive vs Arrive, sorry missed this one last time.

Reply:
:(